mule/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${PYTHON} setup.py install --single-version-externally-managed --record record.txt
```

Comment:
In this file you have added a line which will call the setup.py from the top level of your module repository, but I have just had a little check and noticed that there isn't a setup.py there.  You need to add one of these for the build to work.